### ISSUE TITLE
Updated OMB Results in Never-Seen Errors

### ISSUE THREAD

**Issue by mahyarmirrashed** [ohmybash/oh-my-bash#283]
When I botted my RPi 4 running headless Raspbian, there was an option to update Oh My Bash. As soon as I did that, every time I log in, I am greeted by these very annoying errors:

```txt
oh-my-bash (package_require): package 'plugin:git' not found.
oh-my-bash (package_require): package 'plugin:bashmarks' not found.
oh-my-bash (package_require): package 'alias:general' not found.
```

Google searching showed results for similar errors with Oh My ZSH but their proposed solutions did not work when modifying the `.bashrc` file in the same way.

**Comment by kjy5** [comment]
Just discovered `oh-my-bash` for the first time and am also experiencing this same issue 😕

**Comment by akinomyoga** [comment]
OK, this must be related to the latest push. I'll take a look.

**Comment by akinomyoga** [comment]
Really sorry for the inconvenience. It seems I was loading a different version of `oh-my-bash` in testing the new commits. I have pushed a fix 028d253.

**Comment by akinomyoga** [comment]
@mahyarmirrashed @kjy5 Could you try `upgrade_oh_my_bash` to see if the problem is solved?

```bash
$ upgrade_oh_my_bash
```

**Comment by kjy5** [comment]
Just did seconds before you sent this. Seems to be working again. Thanks for the quick turnaround!

**Comment by akinomyoga** [comment]
OK, thank you for the check! I'll keep this issue open for a while so that other users can find it.

**Comment by mahyarmirrashed** [comment]
> @mahyarmirrashed @kjy5 Could you try `upgrade_oh_my_bash` to see if the problem is solved?
> 
> ```shell
> $ upgrade_oh_my_bash
> ```

Fixed! Thanks for the quick reply. Much obliged and appreciated!

**Comment by mahyarmirrashed** [comment]
Feel free to close this ticket at your discretion.

**Comment by akinomyoga** [comment]
OK! Thank you two again for reporting the issue!

**Comment by akinomyoga** [comment]
There's an additional fix 2c20b7b.
